content/TrueCommand/UserGuide/FleetDashboard.md

```diff
@@ -2,6 +2,72 @@
 title: Fleet Dashboard
 description: "Overview and usage notes for the Fleet Dashboard Screen."
 weight: 15
+tags:
+- tcsystem
+- tcdashboard
+- tcalerts
+- tcupdate
 ---
 
-Placeholder
+In addition to the main TrueCommand [**Dashboard**]({{< relref "/TrueCommand/UserGuide/Dashboard.md" >}}) that shows individual system cards with performance  information for each system managed by TrueCcommand, release 3.0 introduces a new **Fleet Dashboard** with new function widgets (system cards).
+
+![FleetDashboard](/images/TrueCommand/Dashboard/FleetDashboard.png "Fleet Dashboard")
+
+The **Fleet Dashboard** provides an overview of system information organzed into six widgets (information card):
+
+* [Systems](#systems-widget) - Lists each system managed by TrueCommand.
+* [Alerts](#alerts-widget) - Lists alerts detected on systems managed by TrueCommand.
+* [Pools](#pools-widget) - Lists all pools detected on systems managed by TrueCommand.
+* [Shares](#shares-widget) - Lists all shares (SMB, NFS) detected on systems managed by TrueCommand.
+* [Data Protection](#data-protection-widget) - Lists data protection tasks configured on TrueNAS systems and detected by TrueCommand.
+* [S.M.A.R.T. Results](#smart-results-widget) - Shows the number of S.M.A.R.T. tests completed on TrueNAS systems and detected by TrueCommand.
+
+To view more information on shares, replication, pools, and alerts, go to the setting menu option an open the related system or alert screen.
+
+## Systems Widget
+
+The **Systems** widget lists all systems added to and managed by TrueCommand. 
+
+![FleetDashboardSystemsWidget](/images/TrueCommand/Dashboard/FleetDashboardSystemsWidget.png "Fleet Dashboard Systems Widget")
+
+The card includes the **Systems** button that opens the [**New System**]() screen, and the [system options menu](#system-options-menu) that provides access to other functions available to each system managed by the TrueCommand interface.
+
+To access system details and management functions, click the gear gear <i class="material-icons" aria-hidden="true" title="Settings">settings</i> icon, then select either **System Inventory**, or if an administration user, **Systems** to open those screens.
```

Comment:
Is this accurate? I'm not seeing a gear in the screenshot?

Reply:
I replaced the icon with one that works.